src/fastoad/notebooks/02_explanation/04_OpenMDAO.ipynb

```diff
@@ -0,0 +1,1010 @@
+{
```

Comment:
<blockquote>It can sometime happen that, when setting up a group, ...</blockquote><p>I'd write "sometime<strong>s</strong>", but we may also write "It may happen that, when setting up a group, ..."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fast-aircraft-design/FAST-OAD/pull/426/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
Changed in dfe233f7